sci-mathematics/xmds/xmds-2.2.2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit distutils-r1
+
+DESCRIPTION="XMDS - The eXtensible Multi-Dimensional Simulator"
+HOMEPAGE="http://www.xmds.org"
+SRC_URI="mirror://sourceforge/xmds/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples mpi"
+
+DEPEND="dev-python/cheetah[${PYTHON_USEDEP}]"
+RDEPEND=">=sci-libs/fftw-3.3.1[mpi?]
+		mpi? ( virtual/mpi )
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/pyparsing[${PYTHON_USEDEP}]
+		dev-python/lxml[${PYTHON_USEDEP}]
+		dev-python/mpmath[${PYTHON_USEDEP}]
+		dev-python/h5py[${PYTHON_USEDEP}]
+		sci-libs/atlas
+		sci-libs/hdf5
+		sci-libs/gsl"
+
+#virtual/cblas
+
+python_install_all() {
+	use doc && HTML_DOCS+=( documentation/. )
+	if use examples; then
+		insinto "/usr/share/doc/${PF}"
+		docompress -x "/usr/share/doc/${PF}/examples"
+		doins -r examples
```

Comment:
if the folder name as the destination, just use `dodoc -r` instead of `insinto` + `doins`